code/game/gamemodes/cult/runes.dm

```diff
@@ -971,27 +788,44 @@ var/list/teleport_other_runes = list()
 	icon_state = "6"
 	color = rgb(255, 0, 0)
 
-/obj/effect/rune/manifest/invoke(mob/living/user)
+/obj/effect/rune/manifest/New(loc)
+	..()
+	notify_ghosts("Manifest rune created in [get_area(src)].", 'sound/effects/ghost2.ogg', source = src)
+
+/obj/effect/rune/manifest/can_invoke(mob/living/user)
 	if(!(user in get_turf(src)))
 		user << "<span class='cultitalic'>You must be standing on [src]!</span>"
 		fail_invoke()
 		log_game("Manifest rune failed - user not standing on rune")
-		return
+		return 0
 	var/list/ghosts_on_rune = list()
 	for(var/mob/dead/observer/O in get_turf(src))
 		if(O.client && !jobban_isbanned(O, ROLE_CULTIST))
-			ghosts_on_rune.Add(O)
+			ghosts_on_rune |= O
 	if(!ghosts_on_rune.len)
 		user << "<span class='cultitalic'>There are no spirits near [src]!</span>"
 		fail_invoke()
 		log_game("Manifest rune failed - no nearby ghosts")
-		return
+		return 0
+	return ..()
+
+/obj/effect/rune/manifest/invoke(mob/living/user)
+	var/list/ghosts_on_rune = list()
+	for(var/mob/dead/observer/O in get_turf(src))
+		if(O.client && !jobban_isbanned(O, ROLE_CULTIST))
+			ghosts_on_rune |= O
 	var/mob/dead/observer/ghost_to_spawn = pick(ghosts_on_rune)
 	var/mob/living/carbon/human/new_human = new(get_turf(src))
 	new_human.real_name = ghost_to_spawn.real_name
 	new_human.alpha = 150 //Makes them translucent
 	visible_message("<span class='warning'>A cloud of red mist forms above [src], and from within steps... a man.</span>")
 	user << "<span class='cultitalic'>Your blood begins flowing into [src]. You must remain in place and conscious to maintain the forms of those summoned. This will hurt you slowly but surely...</span>"
+	for(get_turf(src))
+		var/obj/machinery/shield/N = new(B)
+		N.name = "Invoker's Shield"
+		N.desc = "A weak shield summoned by cultists to protect them while they carry out delicate rituals"
+		N.color = "red"
+		N.health = 20
```

Comment:
OK based on what you said the intent was in IRC:
remove the for statement and unindent all this bit. Replace `new(B)` at line 824 with `new(get_turf(src))`